runtime/compiler/env/j9methodServer.cpp

```diff
@@ -1041,16 +1041,12 @@ TR_ResolvedJ9JITaaSServerMethod::isInlineable(TR::Compilation *comp)
       }
    }
 
-bool
```

Comment:
We may still needs this code when caching is disabled. 
When caching is enabled we need to check the cache. Note that the structure of the cache at the server is different than the structure of the IProfiler at the client, thus you cannot simply call `TR_ResolvedJ9Method::isWarmCallGraphTooBig`

Reply:
Same thing as below, since it is calling `profilingSample`, it would automatically fetch an entries from the client/server as needed.

Reply:
correct